qemu/tests/cfg/uefishell.cfg

```diff
@@ -0,0 +1,22 @@
+- uefishell:
+    only ovmf
```

Comment:
Do you need to specify 'only q35'?

Reply:
@YanhuiMa I am not sure, i440fx should support it as well, @xiangchunfu could you help confirm? Thanks.

Reply:
@jingzhao84 Thanks, @YanhuiMa Then that should be limited in avocado-vt, not here.